ndk-build/src/apk.rs

```diff
@@ -9,6 +9,24 @@ use std::fs;
 use std::path::{Path, PathBuf};
 use std::process::Command;
 
+#[derive(Debug, Copy, Clone, PartialEq, Eq, serde::Deserialize)]
+#[serde(rename_all = "snake_case")]
+pub enum StripConfig {
+    /// Does not treat debug symbols especially
```

Comment:
`especially` is an adverb...

Reply:
Maybe more clear to explain (here and for the other options, also in the README) that Rust's stripping may also have an affect (in the sense that `Split` won't have anything to work with anymore).